pipelines/pr.yaml

```diff
@@ -23,13 +25,37 @@ jobs:
       # the specific flavors below that are experiencing failures in CI (for example,
       # if we're actively working on features that have a lot of churn on underlying
       # WMR APIs, buildUWPArm would be a good candidate to re-enable)
+      buildStandalone: true
       buildUWPX86: false
       buildUWPArm: false
       buildUWPDotNet: false
       # For mrtk_pr builds, validation scripts (code and docs) should be scoped to only
       # the set of changed files, so that we can save some time. There's no need to
       # check unchanged files for code style/doc style violations.
       scopedValidation: true
+      # This particular job shouldn't run tests (they happen in PRValidationTest)
+      runTests: false
       UnityVersion: $(Unity2018Version)
 
   - template: templates/end.yml
+- job: PRValidationTest
+  timeoutInMinutes: 20
+  pool:
+    name: On-Prem Unity
+    demands:
+    - Unity2018.4.6f1
+    - COG-UnityCache-WUS2-01
+    - SDK_18362 -equals TRUE
+  steps:
+  - template: templates/common.yml
+    parameters:
+      # The only task that this job should do is run the tests.
+      # Building is handled by PRValidationBuild.
+      runTests: true
```

Comment:
Doesn't testing actually depend on the build? I mean, we're probably building twice as attempt to run the tests compiles the x86 configuration.
Also, the common.yml template has this note:
# Build Standalone x86.
# This must happen before tests run, because if the build fails and tests attempt
# to get run, Unity will hang showing a dialog (even when in batch mode).
# This is the fastest build, since it doesn't produce an AppX.